### ISSUE TITLE
Deprecated warn message

### ISSUE THREAD

**Issue by wbiker** [Raku/tap-harness6#44]
Unfortunately, there is this message shown in each test run:
```
Saw 1 occurrence of deprecated code.
================================================================================
Method wait (from TAP::Result) seen at:
  /usr/share/perl6/vendor/sources/9E59A66A5978067EEE04BC039416C2CBDD2F5F92 (TAP), line 148
Please use exitcode and/or signal methods (status is to be removed in 2022.06) instead.
--------------------------------------------------------------------------------
Please contact the author to have these occurrences of deprecated code
adapted, so that this message will disappear!
```
The code line in question is 
https://github.com/Raku/tap-harness6/blob/master/lib/TAP.pm#L148

Please, would someone be so nice and take a look?
Thank you!

**Comment by MasterDuke17** [comment]
I believe this was fixed in https://github.com/Raku/tap-harness6/pull/43, but maybe there hasn't been a new release yet.

**Comment by wbiker** [comment]
I just realized version 1.0 is one year old, so I hope a new release will come soon.

**Comment by Leont** [comment]
Fixed in the new version

**Comment by CIAvash** [comment]
Now I'm getting a deprecated message for method [exit-failed](https://github.com/Raku/tap-harness6/blob/388163cfdec7804049770c0cf6d9ebc1a8809585/lib/TAP.pm#L111), it seems to use `status` as well.

Should I open another issue? Or you could reopen this issue.

**Comment by Leont** [comment]
Fixed that one in 0.2.1
